views/report.hbs

```diff
@@ -27,7 +27,12 @@
         {{/if}}
       </li>
     <li class="report-section__list-item">
-      <img class="report-section__icon" src="/images/x-mark.svg" alt=""><p class="report-section__prereq">Codewars: authored 1+ Kata</p>
+      {{#if codewars.hasAuthored}}
```

Comment:
Can we display the error message if there is an error retrieving the data? It's a little messy in handlebars but this can be done with unless, check lines 18-22 ❤️